pronto/api/entry/go.py

```diff
@@ -258,7 +258,7 @@ def get_term_constraints(accession, term_id):
             if constraint["type"] == "never_in_taxon":
                 if left_num <= taxon_left_num <= right_num:
                     is_ok = False
-                else:
+
```

Comment:
```suggestion
                else:
```

The `matches` property is not used to record violations, but constraint compliances. The `else` is required to increment the number of compliances of the constraint.